kotlin-language-server/build.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
+  ext.kotlin_version = '1.1.0'
+
+  repositories {
+//    mavenCentral()
+    // TODO: ...
+    jcenter()
```

Comment:
TODO: Is it possible to get the Shadow Gradle plugin from Maven Central (which as I understand it has higher standards for validation/signing than JCenter)?